src/Microsoft.Diagnostics.ExtensionCommands/ExtensionMethodHelpers.cs

```diff
@@ -32,6 +33,8 @@ public static string ConvertToHumanReadable(this double totalBytes)
             return $"{updated:0.00}gb";
         }
 
+        public static string ToSignedHexString(this int offset) => offset < 0 ? $"-{Math.Abs(offset):x}" : offset.ToString("x");
```

Comment:
```suggestion
        public static string ToSignedHexString(this int offset) => offset < 0 ? $"-{Math.Abs(offset):x2}" : offset.ToString("x2");
```
To get both nibbles even for single digits numbers? Here as this method is quite universal (not limited to some expected range of `offset`).
See [sharplab example](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEUCuA7AHwAEAmABgFgAoUgRmuoEs8MACANwEMAbVgXlYoA3NSK0AnAAoAJACIyCAN5duIBAF9ZAShE0JM+UpVqSmnUA=) for difference.